lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2607,6 +2607,72 @@ def visit_computed_column(self, generated):
             text += " PERSISTED"
         return text
 
+    def visit_set_table_comment(self, create):
+        schema = self.preparer.schema_for_object(create.element)
+        schema_name = schema if schema else self.dialect.default_schema_name
+        return """
+           execute sp_addextendedproperty 'MS_Description', {0}, 'schema', {1}, 'table', {2};
```

Comment:
@zzzeek a ddl cannot specify parameters rights? I believe this execute sql should support them, but we cannot pass parameters using a ddl at the moment, right?

Reply:
In the other dialects instances of visit_set_table_comment use string formatting.